src/inspector/models/score/scoreaccessibilitysettingsmodel.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <QObject>
+
+#include "models/abstractinspectormodel.h"
+#include "context/iglobalcontext.h"
+#include "global/iglobalconfiguration.h"
+#include "engraving/iengravingconfiguration.h"
+#include "engraving/types/types.h"
+
+namespace mu::inspector {
+class ScoreAccessibilitySettingsModel : public AbstractInspectorModel
+{
+    Q_OBJECT
+    INJECT(mu::context::IGlobalContext, globalContext)
+    INJECT(muse::IGlobalConfiguration, globalConfiguration);
+    INJECT(engraving::IEngravingConfiguration, engravingConfiguration);
+
+public:
```

Comment:
Under `INJECT` and above `public`, you should have these lines:
```C++
Q_PROPERTY(QVariantList possibleScoreStylePresets READ possibleScoreStylePresets NOTIFY possibleScoreStylePresetsChanged);
Q_PROPERTY(int scoreStylePresetIndex READ scoreStylePresetIndex WRITE setScoreStylePresetIndex NOTIFY scoreStylePresetIndexChanged);
```
This declares two properties that you can use in the QML as:

- `model.possibleScoreStylePresets`
- `model.scoreStylePresetIndex`

For this to work, the class also needs to have these functions, as declared in the `Q_PROPERTY()` macros:

```C++
public:
    QVariantList possibleScoreStylePreset() const; // READ: not Q_INVOKABLE now
    int scoreStylePresetIndex() const;             // READ: not Q_INVOKABLE now
    int setScoreStylePresetIndex(int index);       // WRITE: replaces loadStyle(int preset)

signals:
    void possibleScoreStylePresetsChanged(); // NOTIFY: replaces accessibilityStyleEditedChanged()
    void scoreStylePresetIndexChanged();     // NOTIFY
```

Note: There's no `WRITE` function for `possibleScoreStylePreset` because it's a read-only property.

Now, assuming you're emitting (with `emit`) those signals in the right places, if you set `currentIndex: model.scoreStylePresetIndex` in the QML, Qt will use the `READ` and `NOTIFY` functions to make sure that `currentIndex` always stays in sync with `model.scoreStylePresetIndex`.